src/test/java/de/retest/recheck/review/GlobalIgnoreApplierTest.java

```diff
@@ -1,46 +1,93 @@
 import static de.retest.recheck.ui.Path.fromString;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.Mockito.mock;
 
+import java.nio.file.Path;
+import java.util.Optional;
+
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
 
+import de.retest.recheck.ignore.RecheckIgnoreLocator;
 import de.retest.recheck.ui.descriptors.Element;
 import de.retest.recheck.ui.descriptors.IdentifyingAttributes;
 import de.retest.recheck.ui.descriptors.MutableAttributes;
 import de.retest.recheck.ui.diff.AttributeDifference;
 
 class GlobalIgnoreApplierTest {
 
-	private static class some {
-	}
+	private static class some {}
+
+	@TempDir
+	Path base;
 
 	@Test
 	void ignoreElement_should_add_filter() {
 		final GlobalIgnoreApplier cut = GlobalIgnoreApplier.create( getInstance() );
 
-		final Element element = Element.create( "myRetestId", mock( Element.class ),
-				IdentifyingAttributes.create( fromString( "/other[1]/some[1]" ), some.class ),
-				new MutableAttributes().immutable() );
+		cut.ignoreElement( createFake() );
 
-		cut.ignoreElement( element );
+		assertThat( unwrap( cut.persist().getIgnores() ).toString() ).isEqualTo( "[matcher: retestid=myRetestId]" );
+	}
 
-		assertThat( cut.persist().getIgnores().toString() ).isEqualTo( "[matcher: retestid=myRetestId]" );
+	Element createFake() {
+		return Element.create( "myRetestId", mock( Element.class ),
+				IdentifyingAttributes.create( fromString( "/other[1]/some[1]" ), some.class ),
+				new MutableAttributes().immutable() );
 	}
 
 	@Test
 	void ignoreAttribute_should_add_filter() {
 		final GlobalIgnoreApplier cut = GlobalIgnoreApplier.create( getInstance() );
 
-		final Element element = Element.create( "myRetestId", mock( Element.class ),
-				IdentifyingAttributes.create( fromString( "/other[1]/some[1]" ), some.class ),
-				new MutableAttributes().immutable() );
-
-		cut.ignoreAttribute( element, new AttributeDifference( "myAttribute", "expected", "actual" ) );
+		cut.ignoreAttribute( createFake(), new AttributeDifference( "myAttribute", "expected", "actual" ) );
 
-		assertThat( cut.persist().getIgnores().toString() )
+		assertThat( unwrap( cut.persist().getIgnores() ).toString() )
 				.isEqualTo( "[matcher: retestid=myRetestId, attribute=myAttribute]" );
 	}
+
+	@Test
+	void project_should_be_preferred_over_userhome() {
+		final RecheckIgnoreLocator locateNonExistentUserHome = new RecheckIgnoreLocator() {
+			@Override
+			public Optional<Path> getProjectIgnoreFile() {
+				return Optional.of( base.resolve( "project/recheck.ignore" ) );
+			}
+
+			@Override
+			public Path getUserIgnoreFile() {
+				return base.resolve( "userhome/recheck.ignore" );
+			}
+		};
+		final GlobalIgnoreApplier cut = GlobalIgnoreApplier.create( getInstance(), locateNonExistentUserHome );
+
+		cut.ignoreElement( createFake() );
+
+		assertThat( cut.persist().getIgnores().toString() ).contains( "project" );
```

Comment:
If you changed `GlobalIgnoreApplier` `List<Filter>` to `List<PersistFilter>` you could more elegantly check for the `Path`.